pkg/render/manager.go

```diff
@@ -555,6 +579,46 @@ func (c *managerComponent) managerProxyContainer() corev1.Container {
 	}
 }
 
+// managerContainer returns the manager container.
+func (c *managerComponent) managerPacketCaptureContainer() corev1.Container {
+	var volumeMounts []corev1.VolumeMount
+	if c.managementCluster != nil {
+		volumeMounts = append(volumeMounts, corev1.VolumeMount{Name: ManagerInternalTLSSecretCertName, MountPath: "/manager-tls", ReadOnly: true})
+	}
+
+	env := []v1.EnvVar{
+		{Name: "PACKETCAPTURE_API_LOG_LEVEL", Value: "Info"},
+	}
+
+	if c.keyValidatorConfig != nil {
+		env = append(env, c.keyValidatorConfig.RequiredEnv("PACKETCAPTURE_API")...)
+		volumeMounts = append(volumeMounts, c.keyValidatorConfig.RequiredVolumeMounts()...)
+	}
+
+	return corev1.Container{
+		Name:            "tigera-packetcapture",
```

Comment:
Is there not a more specific name we could give this? like `tigera-packetcapture-server` or something that indicates it's purpose?

The container itself, I assume, is not a packet capture :D